bower.json

```diff
@@ -2,7 +2,7 @@
     "name": "adapt-hotgrid",
     "repository": "git://github.com/cgkineo/adapt-hotgrid",
     "framework": "^2.0.0",
-    "version": "2.0.0",
+    "version": "3.0.0",
```

Comment:
Ah, it's just we were trying to keep the major version number for plugins in line with that of Adapt